.github/ISSUE_TEMPLATE/endpoint_request.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Endpoint Request
+about: Suggest a new API endpoint for this project
+title: ''
+labels: enhancement endpoint
+assignees: ''
+
+---
+
+### Basic Info
+
+- **Is this for a new endpoint, a new collection, or an update to an existing endpoint?**
+- **Is this a breaking change?** 
+- **What endpoint(s) are you looking for?** What new endpoint(s) do you want? Or,
+if this is for an existing endpoint, which one? If you are not sure what endpoints
+apply, describe the type of data being served below.
```

Comment:
```suggestion
apply, describe what type of data you would like umdio to provide below.
```

Reply:
or stick with `the` instead of `what` if you prefer, I don't really care on second thought